tests/main/debug-migrate-home/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Check the debug command migrate-home migrates snaps' homes
+
+environment:
+    NAME: test-snapd-tools
```

Comment:
I would use test-snapd-tools instead of using a var, so it is more consistent with other tests